Firestore/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
+# Unreleased
+- [fixed] The SDK no longer fully deserializes queries during garbage,
```

Comment:
Extra comma after garbage.

Also consider rephrasing to be less technically detailed. Something like: "Fixed a crash that could happen when the SDK encountered invalid data during garbage collection.

Reply:
Updated.